.travis.yml

```diff
@@ -39,7 +39,7 @@ before_script:
 script:
   - mkdir build && cd build
   - python ../utils/linuxBuild.py --profile ${DXC_BUILD_TYPE} -GNinja --cc ${CC} --cxx ${CXX} .. .
-  - ninja dxc
+  - ninja dxc && ninja clang-spirv-tests
   - ./bin/dxc --help
   - ./bin/dxc -T ps_6_0 ../tools/clang/test/CodeGenSPIRV/passthru-ps.hlsl2spv
-  - ./bin/dxc -T ps_6_0 ../tools/clang/test/CodeGenSPIRV/passthru-ps.hlsl2spv -spirv
+  - ./bin/clang-spirv-tests --spirv-test-root ../tools/clang/test/CodeGenSPIRV/
```

Comment:
Can kill the above test now. :)

Reply:
I'd actually rather keep these two. Since we cannot run the Microsoft tests on Linux (due to lack of TAEF), running the help and running a simple compile to generate DXIL is worth something.
It shows that we are able to invoke the dxc executable (rather than the gtest executable) and also shows that we can generate DXIL for a simple test.
For example it is possible that we accidentally break the dxc executable on Linux, but the SPIRV codegen tests might still pass.

Reply:
Oh, sorry, my bad. I thought they are still for SPIR-V CodeGen. But they are not. Reasonable.